### ISSUE TITLE
Create a direct obligation

### ISSUE THREAD

**Issue by jonnalley** [HHS/OPRE-OPS#790]
### User Story
As an OPRE staff member, I want to create a new direct obligation so that I can facilitate research or program work in support of a project goal or topic.

### Acceptance Criteria
- [ ] User can provide required attributes for and create an agreement (of type direct obligation) with all required attributes that is persistently stored
- [ ] User should see an indication of some kind that the creation was successful
- [ ] User should not be able to create other agreement types in this story

-----------------------------------------

#### UX
- [ ] Conduct research as needed
- [ ] Develop a user workflow / process map of an agreement, if needed
- [ ] Explore design options
- [ ] Share designs with stakeholders and other OPRE team members for feedback
- [ ] Refine the design based on feedback
- [ ] Provide design assets with annotations to developers

#### Dev
- [ ] Update backend model
- [ ] Update/add backend business logic
- [ ] Create frontend components
- [ ] Update breadcrumbs/navigation as necessary

### Definition of Done Checklist
- [ ] Usability validated
- [ ] UI works as designed
- [ ] OESA: Code refactored for clarity
- [ ] OESA: Dependency rules followed
- [ ] Automated unit tests updated and passed
- [ ] Automated integration tests updated and passed
- [ ] Automated quality tests updated and passed
- [ ] Automated load tests updated and passed
- [ ] Automated a11y tests updated and passed
- [ ] Automated security tests updated and passed
- [ ] 90%+ Code coverage achieved
- [ ] PR(s) have been merged to main
- [ ] Design/tech debt eliminated
- [ ] Build process updated
- [ ] Documentation updated or added
- [ ] Feature flags/toggles created

**Comment by xlorepdarkhelm** [comment]
Do we know what fields will be needed for Direct Obligation yet?

**Comment by jonnalley** [comment]
@xlorepdarkhelm no, we only have info on contracts so far. Haven't explored any other agreement type
